server/src/test/java/io/crate/integrationtests/RestSQLActionIntegrationTest.java

```diff
@@ -227,4 +227,23 @@ public void test_failing_delete_bulk_response_errors_are_truncated() throws Exce
         // The last error message must not be available in the response
         assertThat(response.body()).contains("{\"rowcount\":-2}");
     }
+
+    /**
+     * https://github.com/crate/crate/issues/18066
+     */
+    @Test
+    public void test_response_formatting_when_single_arg_bulk_operation_threw_runtime_error() throws Exception {
+        execute("CREATE TABLE doc.insert_test (id INT PRIMARY KEY) CLUSTERED INTO 1 SHARDS");
+        var body = """
+            {
+              "stmt": "INSERT INTO doc.insert_test(id) VALUES(?)",
+              "bulk_args": [[1]]
+            }
+            """;
+        post(body);
+        var response = post(body);
+        assertThat(response.statusCode()).isEqualTo(200);
```

Comment:
Looking at https://github.com/crate/crate/blob/855f4d9d710de44cee744fa7411170accfd2632f/server/src/test/java/io/crate/integrationtests/RestSQLActionIntegrationTest.java#L147, returning `200` is OK.